express/index.cjs

```diff
@@ -458,26 +444,27 @@ app.put('/reset-password', async (req, res) => {
 
     if (userType === 'seeker') {
       query = 'UPDATE Seeker SET user_pass = :hash WHERE seeker_id = :userId';
-      params = { hash, userType };
+      params = { hash, userIdHex };
     } else {
       query = 'UPDATE Employer SET user_pass = :hash WHERE employer_id = :userId';
-      params = { hash, userType };
+      params = { hash, userIdHex };
     }
 
     console.log(`Executing query: ${query} with params: ${JSON.stringify(params)}`);
 
     await req.db.query(query, params);
 
-    writer.write(`${setTimestamp(newTime)} | [Password reset] | reset-password | [success] | [Password successfully reset] | 1 attempt + ${userType}\n`);
+    writer.write(`${setTimestamp(newTime)} | [Password reset] | reset-password | [success] | [Password successfully reset] | 1 attempt + ${userId}\n`);
```

Comment:
Error message: ReferenceError: userId is not defined

Reply:
Where is the definition? I see userIdHex and userType, but not userId

Reply:
userId is supposed to be userIdHex, but I'm trying to pass the params from the helper.js function to the endpoints instead